components/applications-service/pkg/storage/postgres/service_groups.go

```diff
@@ -222,22 +229,100 @@ func (db *Postgres) GetServiceGroups(
 	return sgDisplay, nil
 }
 
-// ReleaseString returns the release string of the service group, our criteria are:
+// ReleaseString returns the release string of the service group.
+//
+// Example: From the package_ident:       'core/redis/0.1.0/20200101000000'
+//          The release string should be: '0.1.0/20200101000000'
+//
+// Criterias:
```

Comment:
The spellchecker is complaining about this, since "criteria" is already plural :)

Reply:
Will change!